Code/Editor/TrackView/SequenceBatchRenderDialog.cpp

```diff
@@ -1215,6 +1216,18 @@ void CSequenceBatchRenderDialog::OnKickIdleTimout()
         // All done with our custom OnKickIdle, restore editor idle.
         SetEnableEditorIdleProcessing(true);
     }
+
+    //When we disable the editor idle processing. system tick is no longer invoked.
+    //so we call it here to ensure rendering + other systems are updated.
```

Comment:
uber nit: Just to be consistent with other comments in the file can we leave a space between the `// ` and the comment 😝 (you can ignore me but just wanted to mention it, comment is definitely helpful though! 👍)